spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/TaskConfiguration.java

```diff
@@ -188,6 +191,23 @@ public TaskRepositoryInitializer taskRepositoryInitializerForDB(DataSource dataS
 			return taskRepositoryInitializer;
 		}
 
+		@Bean
+		@DependsOn({"batchRepositoryInitializerForDefaultDB", "taskRepositoryInitializerForDB"})
+		public AbstractDatabaseInitializer taskBatchRdbmsIndexInitializer(DataSource dataSource, ResourceLoader resourceLoader) {
```

Comment:
It is not an abstract class, so the name should be different.  Also, not quite a fan of defining these classes inside an `@Bean` definition, too much coupling.  I'll rename and move to a top level class.